Z/zimg/build_tarballs.jl

```diff
@@ -22,7 +22,7 @@ install_license COPYING
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = filter!(p -> arch(p) != "aarch64", supported_platforms(; experimental=true))
+platforms = supported_platforms(; exclude=p -> arch(p) == "aarch64")
```

Comment:
Did you try building for aarch64 to see whether the problem we had before was solved?

Reply:
Yep, lots of errors in files specifically for arm builds, see: https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=16984&view=logs&jobId=990d1ed3-c5c4-5990-c40c-9dc4afe8bc4e&j=990d1ed3-c5c4-5990-c40c-9dc4afe8bc4e&t=1afabce2-94fc-5ea8-780d-7c677906c755